### PR TITLE
Support for clamps. Minor features changes for ESD and Res

### PR DESCRIPTION
**Pull request by boblinchuan**
- Clamp cell layout and schematic
- ESD static: top metal layer control
- Res: connect=True for standalone LVS

### CODE REVIEW

**Comment by ayan-biswas** [comment]
Looks good to me. But please remember to also add these changes to the without_OA branch as well.

**Review by ayan-biswas — APPROVED** [review]
Looks good to me. But please remember to add these changes to the "without_OA" branch as well. Also rename the PR to something more meaningful like "adding clamps; editing esd and res".

File: src/xbase/schematic/netlist_info/clamp_static.symbol.yaml [hunk 1]
```diff
@@ -0,0 +1,175 @@
+lib_name: xbase
+cell_name: clamp_static
+view_name: symbol
+bbox:
```
Comment (ayan-biswas) [inline src/xbase/schematic/netlist_info/clamp_static.symbol.yaml:4]: Did you ensure that the lower left coordinate of the red box in the symbol is at (0, 0), and all pins are on grid?

**Review by hyunjaekwon — APPROVED** [review]

File: src/xbase/schematic/netlist_info/clamp_static.symbol.yaml (hunk 1 above)
Comment (boblinchuan) [reply src/xbase/schematic/netlist_info/clamp_static.symbol.yaml:4]: Looks good to me. I did push one more update to draw a more useful symbol.

**Review by FeliciaG — APPROVED** [review]

**Comment by boblinchuan** [comment]
Now that `develop` and `without_OA` branches are merged, I can merge this PR just once :).
The features in this PR here are all additive and so can't affect negatively affect / break anything.
I owe a longer explanation of my `sch_template` class. That's a TODO.

**Pull request merged**